core/bot_service.go

```diff
@@ -0,0 +1,236 @@
+package core
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"path"
+
+	"github.com/mr-tron/base58/base58"
+	shared "github.com/textileio/go-textile-core/bots"
+	"github.com/textileio/go-textile/crypto"
+	ipfs "github.com/textileio/go-textile/ipfs"
+)
+
+// BotIpfsHandler is the ipfs interface shared with a bot. It requires Get and Add
+type BotIpfsHandler struct {
```

Comment:
does this implement and interface in go-textile-core?

Reply:
it does yeah, https://github.com/textileio/go-textile-core/blob/master/bots/interface.go#L29, i've updated my doc string in upcoming push